core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -85,6 +88,12 @@
 public class ApiListener extends PushingListenerAdapter implements HasPhysicalDestination, ReceiverAware<Message> {
 
 	private static final Pattern VALID_URI_PATTERN_RE = Pattern.compile("([^/]\\*|\\*[^/\\n])");
+	private static final Pattern URI_PATTERN_VARIABLES_RE = Pattern.compile("/\\{(.+?)}(?=/|$)");
+
+	/**
+	 * These are names that are never allowed as HTTP parameters, because the Frank!Framework sets these names as session variables.
+	 */
+	public static final Set<String> RESERVED_NAMES = Set.of("originalMessage", "ClaimsSet", "allowedMethods", "HttpMethod", "headers", "updateEtag", "apiPrincipal", "uri", "remoteAddr", "authorizationToken", "securityHandler", "servletRequest", "servletResponse", "multipartAttachments");
```

Comment:
This set of reserved names should probably be in the PipeLineSession, although I'm not sure how to best use it there outside the ApiListener.